### DISCUSSION TITLE
Provider data

### DISCUSSION THREAD

**Discussion by rabiddoughnuts** [trakt/trakt-api#408] [Q&A] [answered]
Is there a way to check provider information using the api? I have a huge list of media, and I want to be able to have it formatted the way I want, which I have, but I want to include where I can watch it, but would like to avoid weeks of work adding it all manually, is it possible to get that info using the api or no?

**Comment by rectifyer** [comment, marked as answer]
No, our agreement with JustWatch doesn't allow for that info to be passed through the API currently.

I suggest using the TMDB API to get that specific info. https://developer.themoviedb.org/reference/movie-watch-providers

**Reply by kevincador** [reply]
Hi Justin,

Just wanted to mention that I think this could be really cool to put this behind a VIP paywall. Not only trakt VIP enduser but also a trakt API VIP one. Some feature have a cost, it should at some point make sense for you and us. I'm not saying you should pull a twitter or reddit shenanigan on us but I trust you guys to build something that makes sense for the community ☺️

I also guess it's just a question of negotiation and cost with JustWatch. I've been in contact with them, I know they're not cheap 😅 

Cheers,

Kevin
